robowflex_library/src/robot.cpp

```diff
@@ -82,7 +82,7 @@ bool Robot::initialize(const std::string &urdf_file, const std::string &srdf_fil
             return false;
         }
 
-    initializeInternal();
+    initializeInternal(true, limits_file);
```

Comment:
Why the limits file is not correctly loaded [here ](https://github.com/KavrakiLab/robowflex/pull/293/files#diff-bddaa6633ef7913447fd7abe905b0c15c93ea39adf6251a4ba310fe3cf477fd4R71). If there is a problem with this function, Then I think it should be fixed and not add a custom function inside robot.

Reply:
Because when the underlying moveIt loads the robot, if it has access to the URDF and SRDF files, it skips loading the joint_limits file. Instead it only gets the limits from the URDF. So if we provide all of these three files, hoping to use our custom joint_limits file, moveIt would never read it and it's not used ([see line 105 here](http://docs.ros.org/en/noetic/api/moveit_ros_planning/html/robot__model__loader_8cpp_source.html)).